### PR TITLE
add spanish language

### PR DESCRIPTION
**Pull request by juanqp07**

### CODE REVIEW

**Comment by thedragon40** [comment]
I have added the Spanish language so that the bot reaches the Spanish community

File: bot/es_es/slash_commands/help.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "help",
+    "description": "Obtenga ayuda con Quark",
+    "commandOptions": {
+      "overviewOption": {
+        "name": "descripción general",
```
Comment (thatkev) [inline bot/es_es/slash_commands/help.json]: > `descripción general`

This should be combined into one word.

File: bot/es_es/slash_commands/help.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        "description": "Ver información de ayuda general"
+      },
+      "serverlogOption": {
+        "name": "registro del servidor",
```
Comment (thatkev) [inline bot/es_es/slash_commands/help.json]: > `registro del servidor`

This should be combined into one word.

File: bot/es_es/slash_commands/unban.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "unban",
+    "description": "Desbanear a un usuario",
+    "commandOptions": {
+      "userOption": {
+        "name": "ID de usuario",
```
Comment (thatkev) [inline bot/es_es/slash_commands/unban.json]: > `ID de usuario`

This should be combined into one word. Should be all lowercase.

**Review by thatkev — CHANGES_REQUESTED** [review]
Please review these changes in order for checks to pass.

**Review by thatkev — APPROVED** [review]

**Pull request merged**